Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -192,6 +192,7 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public C
     bool hasPendingBackForwardItem() { return m_hasPendingBackForwardItem; }
 
     WebCore::LayerHostingContextIdentifier layerHostingContextIdentifier() const { return m_layerHostingContextIdentifier; }
+    void setFrameSize(WebCore::IntSize size) { m_frameSize = size; }
```

Comment:
I think this would be more clear if it were like this:

void setRemoteFrameSize(WebCore::IntSize size) { m_remoteFrameSize = size; }